packages/react-component-library/src/components/DatePickerE/DatePickerE.stories.tsx

```diff
@@ -33,17 +33,37 @@ const Template: ComponentStory<typeof DatePickerE> = (args) => (
 
 export const Default = Template.bind({})
 
-export const WithExistingValue = Template.bind({})
-WithExistingValue.storyName = 'With existing value'
-WithExistingValue.args = {
-  startDate: parseISO('2021-12-15'),
+export const WithInitialValue = Template.bind({})
+WithInitialValue.storyName = 'With initial value'
+WithInitialValue.args = {
+  initialStartDate: parseISO('2021-12-15'),
 }
 
+export const WithExternalState: ComponentStory<typeof DatePickerE> = () => {
```

Comment:
I wonder if the Storybook story is superfluous? Albeit necessary to develop the feature.

Reply:
I think they're useful for showing how to do this type of stuff. I'll remove them though as they aren't meant to be 'use buttons in this way' either...